### DISCUSSION TITLE
Automated SSO

### DISCUSSION THREAD

**Discussion by codyshoffner** [requarks/wiki#7375] [Help / Questions] [answered]
I am currently using the Wiki.Js helm chart to automate Wiki deployments. As it stands now, I can configure Wiki.Js in an automated fashion, but I'm stunted by having to switch to ClickOps when it comes to configuring sso. 

Is it possible to automate this step? If not, is there intention in getting this added?

I have found this discussion previously, but it is over 2 years old now. https://github.com/requarks/wiki/discussions/5149

**Comment by NGPixel** [comment, marked as answer]
No plan to automate this. But you could setup your wiki how you like and then take a snapshot of the DB. Use that snapshot as a pre-configured starting point.
